src/charts.ts

```diff
@@ -37,6 +42,27 @@ export interface LineData {
 export async function writeChartToFile(xData: number[], lines: LineData[], fileType: string = 'png', startTimeStamp: number = 0, xAxisLabel: string = 'elapsed(ms)', file: string = null, title: string = null): Promise<Buffer> {
 	xAxisLabel = `${xAxisLabel}, Start time:${toDateTimeString(startTimeStamp)}`;
 	xData = xData.map(x => x - xData[0]);
+	const yAxes: ChartYAxe = {

```

Comment:
Not sure what these changes have to do with moving collection out to separate process. It's usually preferred to try and keep unrelated changes in separate PRs to help reduce review randomization.

Reply:
these have to do with pulling in @types for dependencies. I agree nevertheless on keeping these kinds of changes separate, so will bear that in mind in future.